### PR TITLE
DOC: move enhancements in 1.5 release notes

### PR DESCRIPTION
**Pull request by simonjayhawkins**
@ParfaitG can you confirm these are enhancements and not api breaking changes.

### CODE REVIEW

**Comment by ParfaitG** [comment]
Confirmed. These are extended, new features of `pandas.read_xml` (new IO module introduced in 1.3).

File: doc/source/whatsnew/v1.5.0.rst [hunk 1]
```diff
@@ -294,83 +373,10 @@ upon serialization. (Related issue :issue:`12997`)
...
-    3578762               Page:Black cat 1897 07 v2 n10.pdf/44  104    219649
-    3578763      The History of Tom Jones, a Foundling/Book IX    0  12084291
-    3578764  Page:Shakespeare of Stratford (1926) Yale.djvu/91  104     21450
-
-    [3578765 rows x 3 columns]
-
-
-.. _`lxml's iterparse`: https://lxml.de/3.2/parsing.html#iterparse-and-iterwalk
-.. _`etree's iterparse`: https://docs.python.org/3/library/xml.etree.elementtree.html#xml.etree.ElementTree.iterparse
+api_breaking_change1
```
Comment (datapythonista) [inline doc/source/whatsnew/v1.5.0.rst:378]: Are these just placeholders? Do we need to keep them now that the section has content?

**Review by datapythonista — APPROVED** [review]
Lgtm, just one question, but feel free to merge when you're ready

File: doc/source/whatsnew/v1.5.0.rst (hunk 1 above)
Comment (simonjayhawkins) [reply doc/source/whatsnew/v1.5.0.rst:378]: I see no harm in keeping for now. Hopefully no significant breaking changes in the 1.5 release other than the ones that are considered bug fixes (in a different section). We may need to add something for #47365 (if we don't revert to 1.4.2 behavior) that is more detailed than would fit in the one-liner section.

**Review by mroeschke — APPROVED** [review]

**Pull request merged**